workspaces/liferay-testray-workspace/build.gradle

```diff
@@ -0,0 +1,9 @@
+apply plugin: "com.liferay.poshi.runner"
```

Comment:
hey @michaelhashimoto  this looks like a nice pattern we can identify, a folder named poshi and a poshi.properties.  That may be enough to add the poshi-runner plugin via workspace.  but we can think about that once we have a working pattern.

Reply:
I see, that makes sense, do you want me to look into editing the workspace plugin?  Or did you want to add that functionality?